### PR TITLE
Set P2P_PEER_CANDIDATE_URL properly for node restarts

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Set P2P_PEER_CANDIDATE_URL properly for node restarts
- Tweak tracker health checking logic in incremental deployment

Related issues:
- https://github.com/ainblockchain/ain-blockchain/issues/732
- https://github.com/ainblockchain/ain-blockchain/issues/730

### CODE REVIEW

File: start_node_incremental_gcp.sh [hunk 1]
```diff
@@ -84,20 +84,32 @@ KEYSTORE_DIR=testnet_dev_staging_keys
...
+    else
+        export P2P_PEER_CANDIDATE_URL="https://summer-api.ainetwork.ai/json-rpc"
+    fi
     KEYSTORE_DIR=testnet_prod_keys
 elif [[ $SEASON = 'staging' ]]; then
     export BLOCKCHAIN_CONFIGS_DIR=blockchain-configs/testnet-staging
-    export P2P_PEER_CANDIDATE_URL="http://35.194.139.219:8080/json-rpc"
+    if [[ $NODE_INDEX -gt 4 ]]; then
+        export P2P_PEER_CANDIDATE_URL="http://35.194.139.219:8080/json-rpc"
+    fi
```
Comment (liayoo) [inline start_node_incremental_gcp.sh:106]: question) why are we not using the API servers for the staging & dev?

**Review by liayoo — APPROVED** [review]
LGTM, thanks!

File: start_node_incremental_gcp.sh [hunk 2]
```diff
@@ -84,20 +84,36 @@ KEYSTORE_DIR=testnet_dev_staging_keys
...
+    # NOTE(platfowner): For non-api-servers, the value in the blockchain configs
```
Comment (liayoo) [inline start_node_incremental_gcp.sh:107]: ahha! 👍

File: start_node_incremental_gcp.sh (hunk 1 above)
Comment (platfowner) [reply start_node_incremental_gcp.sh:106]: It has the same reasons as TRACKER_WS_ADDR.
(ref: https://github.com/ainblockchain/ain-blockchain/pull/711/files#r753124622)

**Pull request merged**